critiquebrainz/frontend/templates/recording/entity.html

```diff
@@ -0,0 +1,140 @@
+{% extends 'base.html' %}
+{% from 'macros.html' import show_avg_rating, entity_rate_form, show_external_reviews with context %}
+
+{% block title %}
+  {{ _('Recording "%(name)s" by %(artist)s', name=recording.name, artist=recording['artist-credit-phrase']) }}
+  - CritiqueBrainz
+{% endblock %}
+
+{% block content %}
+<div class="clearfix">
+  <h2 class="pull-left">
+    {% set artist = [] %}
+    {% for credit in recording['artists'] %}
+      {% if credit.name %}
+        {% do artist.append('<a href="%s">'|safe % url_for('artist.entity', mbid=credit.id) ~ credit.name ~ '</a>'|safe) %}
+        {% if credit.join_phrase %}
+          {% do artist.append(credit.join_phrase) %}
+        {% endif %}
+      {% else %}
+        {% do artist.append(credit) %}
+      {% endif %}
+    {% endfor %}
+
+    {{ _('%(recording)s by %(artist)s', recording = recording.name, artist = artist|join()) }}
+  </h2>
+
+  {% if not my_review %}
+    <a id="write-review" href="{{ url_for('review.create', recording=id) }}"
+       role="button" class="btn btn-success pull-right">
+      <span class="glyphicon glyphicon-asterisk"></span> {{ _('Write a review') }}
+    </a>
+  {% else %}
+    <a id="edit-review" href="{{ url_for('review.edit', id=my_review.id) }}"
+       role="button" class="btn btn-primary pull-right">
+      <span class="glyphicon glyphicon-edit"></span> {{ _('Edit my review') }}
+    </a>
+  {% endif %}
+</div>
+
+<div id="recording-details" class="row">
+  <div class="col-md-9">
+    {{ entity_rate_form('recording', 'recording') }}
+    <br/><br/>
+    <h4 style="margin-bottom:0;">{{ _('Reviews') }}</h4>
+    {% if not reviews %}
+      <p class="lead" style="text-align: center;">{{ _('No reviews found') }}</p>
+    {% else %}
+      <table class="table table-condensed table-hover">
+        <thead>
+          <tr>
+            <th></th>
+            <th>{{ _('Published on') }}</th>
+            <th>{{ _('Votes (+/-)') }}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {% for review in reviews %}
+            <tr data-review-id="{{ review.id }}">
+              <td>
+                <a href="{{ url_for('review.entity', id=review.id) }}">
+                  {{ _('by %(reviewer)s', reviewer='<img class="avatar" src="%s&s=16" /> '|safe % review.user.avatar + review.user.display_name) }}
```

Comment:
again, don't put html inside a variable, unless this is a best practise for translation

Reply:
Fixed.